tests/ops/functions/test_map_wires.py

```diff
@@ -0,0 +1,211 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for the qml.map_wires function
+"""
+import pytest
+
+import pennylane as qml
+from pennylane.tape import QuantumTape
+from pennylane.wires import Wires
+
+
+def build_op():
```

Comment:
We can probably make this private

Reply:
If it's in a test file, I don't think that matters too much.